src/github.com/getlantern/flashlight/client/client.go

```diff
@@ -5,18 +5,30 @@ import (
 	"net"
 	"net/http"
 	"reflect"
+	"strings"
 	"sync"
 	"time"
 
 	"github.com/armon/go-socks5"
 	"github.com/getlantern/balancer"
+	"github.com/getlantern/detour"
 	"github.com/getlantern/eventual"
 	"github.com/getlantern/golog"
 )
 
+const (
+	// LanternSpecialDomain is a special domain for use by lantern that gets
+	// resolved to localhost by the proxy
+	LanternSpecialDomain          = "ui.lantern.io"
```

Comment:
Unfortunately I can't use a subdomain of getlantern.org because that domain was configured with HSTS, including for subdomains.  So, Chrome always switches over to an HTTPS connection, which our embedded UI server doesn't support.